docs/fips.md

```diff
@@ -0,0 +1,78 @@
+
+## Prerequisites
+
+To prepare Percona Server for MySQL for FIPS certification, do the following:
+
+* Check that your operating system includes FIPS pre-approved OpenSSL library in version 3.0.x or higher. The following distributions includes FIPS pre-approved OpenSSL library in version 3.0.x or higher:
+
+    * RedHat Enterprise Linux 9 and derivatives
+
+    * Oracle Linux 9
+
+    The following distributions also includes OpenSSL library in version 3.0.x but do not have FIPS-approved crypto provider installed by default (you can build the crypto provider from the source for testing):
+
+    * Debian 12
+
+    * Ubuntu 22.04 Pro (the OpenSSL FIPS 140-3 certification is under implementation)
+
+        !!! note
+            
+            If you enable FIPS on Ubuntu Pro with `$ sudo pro enable fips-updates` and then disable FIPS with `$ sudo pro disable fips-updates`, Percona Server for MySQL may stop operating properly. For example, if you disable FIPS on Ubuntu Pro with `$ sudo pro disable fips-updates` and enable the FIPS mode on Percona Server with `ssl-fips-mode=ON`, Percona Server may not load the SSL certificate.
+
+* Deploy [Percona Server for MySQL from the Pro build](psmysql-pro.md), which is built and tested on operating systems with FIPS pre-approved OpenSSL packages.
+
+## The FIPS mode variables
+
+Percona Server for MySQL uses the same variables and values as MySQL. Percona Server for MySQL enables control of FIPS mode on the server side and the client side:
+
+* The `ssl_fips_mode` system variable shows whether the server operates in FIPS mode. This variable is disabled by default.
+
+    The `ssl_fips_mode` system variable has these values:
+
+    * `0` - disables FIPS mode
+    * `1` - enables FIPS mode
+    * `2` - enables `strict` FIPS mode. This value provides more restrictions than the  `1 ` value. However, MySQL itself does not have any FIPS-specific code other than to specify the FIPS mode value to OpenSSL. The exact behavior of FIPS mode for `1` or `2` depends on the OpenSSL version.
+
+* The `--ssl-fips-mode` client/server option controls whether a given MySQL client operates in FIPS mode. This setting does not change the server setting. This option is disabled by default.
+
+    The `--ssl-fips-mode` client/server option has these values:
+
+    * `OFF` - disables FIPS mode
+    * `ON` - enables FIPS mode
+    * `STRICT` - enables `strict` FIPS mode. This value provides more restrictions than the `ON` value. However, MySQL itself does not have any FIPS-specific code other than to specify the FIPS mode value to OpenSSL. The exact behavior of FIPS mode for `ON` or `STRICT` depends on the OpenSSL version.
```

Comment:
Same as above.
We don't know if any restrictions are applied to the client because that's dependent on the OpenSSL version.